### ISSUE TITLE
I think that there is no need adding not null and unique constraint as they are alredy included in the primary key constraint 

### ISSUE THREAD

**Issue by monicamillad** [hazemessam/cms#1]
https://github.com/hazemessam/cms/blob/ddc43682d565fb175fbaab42a4a2034a98ec88cb/src/main/resources/db/changelog/1723217176967-add-initial-schema.xml#L12C30-L12C40

**Comment by monicamillad** [comment]
will be also applied:

https://github.com/hazemessam/cms/blob/ddc43682d565fb175fbaab42a4a2034a98ec88cb/src/main/resources/db/changelog/1723217176967-add-initial-schema.xml#L38

https://github.com/hazemessam/cms/blob/ddc43682d565fb175fbaab42a4a2034a98ec88cb/src/main/resources/db/changelog/1723217176967-add-initial-schema.xml#L64

https://github.com/hazemessam/cms/blob/ddc43682d565fb175fbaab42a4a2034a98ec88cb/src/main/resources/db/changelog/1723217176967-add-initial-schema.xml#L104

https://github.com/hazemessam/cms/blob/ddc43682d565fb175fbaab42a4a2034a98ec88cb/src/main/resources/db/changelog/1723217176967-add-initial-schema.xml#L135

**Comment by hazemessam** [comment]
@monicamillad You are right actually they have no effect on the DB, but I don't think we can remove them as directly updating an applied change set will lead Liquibase to generate a new checksum different than the one stored in the DB which will lead to a migration error at startup.
